editor/editor_settings.cpp

```diff
@@ -527,6 +527,7 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	// SceneTree
 	_initial_set("docks/scene_tree/start_create_dialog_fully_expanded", false);
 	_initial_set("docks/scene_tree/auto_expand_to_selected", true);
+	_initial_set("docks/scene_tree/center_node_on_reparent", true);
```

Comment:
```suggestion
	_initial_set("docks/scene_tree/center_node_on_reparent", false);
```
New settings that change behaviour should be turned off by default, they should be opt-in